### ISSUE TITLE
Snap to line behavior can cause difficulty in matching ratio

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ratio-and-proportion#284]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/582
This is not exactly a bug, per say, but it tripped me up for long enough before I realized what was happening that it might be worth re-examining the behavior. It is possible to set up one hand so that, to get the correct ratio with the second hand it must be placed near, but not on, a line. If this happens then the snap-to-line behavior can make it frustrating not possible to achieve, without making it immediately clear why the behavior is happening.
**Steps to reproduce**
1. Go to the second screen and set up an easy to work with ratio.
2. Turn on the lines, and move the hands so that they match the ratio with both hands on a line and neither at an extreme end.
3. Turn off the lines
4. Move one of the hands slightly upwards. Just enough that the ratio no longer matches.
5. Turn on the lines, and try to move the second hand to match the target ratio

**Visuals**
![dropsdowntoline](https://user-images.githubusercontent.com/41024075/101963425-ec12ff80-3bcb-11eb-93c4-4dd7d2ef3568.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.77/phet/ratio-and-proportion_all_phet.html
Version: 1.0.0-dev.77 2020-12-08 00:32:10 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
Language: en-US
Window: 1280x658
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by zepumph** [comment]
This is a know an expected problem; a biproduct of the design. I will toss this over to @emily-phet to confirm that we aren't going to do anything for this, or to recommend a potential solution.

**Comment by emily-phet** [comment]
@KatieWoe This is a trade-off we decided to make when adding the snap-to-line feature. Without the snap to line feature, it can be difficult to find 'at ratio" exactly and with the size of the hands etc., people were getting the wrong idea about the ratio relationships when they were slightly off from "at ratio". With the snap-to-line feature, it means that sometimes you snap into or out of ratio unintentionally. 

We decided to add the feature, because it seemed like this helped those just getting familiar with the sim to understand what the relationships were to attend to.
